### PR TITLE
feat(dino-park): making slack be the default account

### PR DESCRIPTION
**Pull request by andrew-sunada**
Updating AccountsMixin to feed list of accounts from 'ENABLED_ACCOUNTS' rather than 'EXTERNAL_ACCOUNTS' in order to maintain specific list order and default value.

### CODE REVIEW

File: src/components/_mixins/AccountsMixin.vue [hunk 1]
```diff
@@ -93,8 +93,8 @@ export default {
   },
   data() {
     return {
-      availableAccounts: [...Object.keys(EXTERNAL_ACCOUNTS)].filter((account) =>
-        ENABLED_ACCOUNTS.includes(account),
+      availableAccounts: ENABLED_ACCOUNTS.filter(
+        (account) => Object.keys(EXTERNAL_ACCOUNTS).indexOf(account) !== -1,
```
Comment (fiji-flo) [inline src/components/_mixins/AccountsMixin.vue]: Why swap this? And if, this way it's easier to read and faster.
```suggestion
        (account) => account in EXTERNAL_ACCOUNTS,
```

Comment (fiji-flo) [reply src/components/_mixins/AccountsMixin.vue]: Sorry, I get the swap here. My bad. Then just the other thing.

Comment (andrew-sunada) [reply src/components/_mixins/AccountsMixin.vue]: Yea, the swap is important, but account in EXTERNAL_ACCOUNTS is definitely the better way to do that.

Comment (andrew-sunada) [reply src/components/_mixins/AccountsMixin.vue]: Completed!

File: src/pages/PageHome.vue [hunk 2]
```diff
@@ -11,81 +11,28 @@
...
-            target="_blank"
-            rel="noopener noreferrer"
-            >Recent Additions</a
-          >
-        </h2>
-        <p>
-          Profile editing is now live with privacy controls and synchronized
-          Workday information. Read more about this and improved people search
-          capabilities in our
-          <a
```
Comment (fiji-flo) [inline src/pages/PageHome.vue]: The inline links here where intentional. Part of it for screen reader, part of it for UX. If feel strong about this, have a conversation with Megan.

File: src/components/ui/Footer.vue [hunk 3]
```diff
@@ -1,59 +1,69 @@
...
+        },
+        {
+          link: this.globalLinks.githubIssues,
+          icon: 'github',
+          label: 'GitHub',
+        },
+        {
+          link: this.globalLinks.feedback,
+          icon: 'envelope',
+          label: 'Helpdesk',
```
Comment (fiji-flo) [inline src/components/ui/Footer.vue]: Part of changing the footer was to remove the help desk. https://jira.mozilla.com/browse/DP-1416 in the comments.

File: src/components/_mixins/LinksMixin.vue [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    return {
+      globalLinks: {
+        feedback: 'https://github.com/mozilla-iam/dino-park-issues/issues',
+        profilePrivacy: 'https://discourse.mozilla.org/t/dinopark-faq/33564',
+        editModeAnnouncement:
+          'https://discourse.mozilla.org/t/edit-mode-announcement/38563',
+        participationGuidelines:
+          'https://www.mozilla.org/en-US/about/governance/policies/participation/',
+        faq: 'https://discourse.mozilla.org/t/dinopark-faq/33564',
+        githubIssues: 'https://github.com/mozilla-iam/dino-park-issues',
```
Comment (fiji-flo) [inline src/components/_mixins/LinksMixin.vue]: I would de-duplicate the links here for consistency.

File: src/App.vue [hunk 5]
```diff
@@ -69,6 +69,7 @@ export default {
   --layerModal: 9;
 
   --shadowCard: 0 0.25em 0.25em 0 rgba(210, 210, 210, 0.5);
+  --shadowCardHover: 0 0 0 0.25rem rgba(210, 210, 210, 0.5);
```
Comment (fiji-flo) [inline src/App.vue]: `rem` → `em` for consistency.

File: src/components/ui/Card.vue [hunk 6]
```diff
@@ -14,10 +14,24 @@ export default {
...
+  padding: 0;
   position: relative;
   border-radius: var(--cardRadius);
 }
+
+.card:hover {
+  box-shadow: var(--shadowCardHover);
+}
+
+.card:hover .card__link h2 {
```
Comment (fiji-flo) [inline src/components/ui/Card.vue]: The bold underline (box-shadow) was done to make it look like the [designs](https://app.zeplin.io/project/5b61ec0e22302659883e6834/screen/5be67996f9d5b53eec29b6ba).

File: src/pages/PageHome.vue [hunk 7]
```diff
@@ -11,81 +11,28 @@
...
-            href="https://discourse.mozilla.org/t/edit-mode-announcement/38563"
-            target="_blank"
-            >Discourse post</a
-          >.
-        </p>
+      <Card v-for="(card, idx) in cards" :key="idx">
+        <a
```
Comment (fiji-flo) [inline src/pages/PageHome.vue]: This makes the highlight look weird (tab through the links). And also changes keyboard navigation behavior (previously the title (h2) and the in text link were select-able.

**Comment by andrew-sunada** [comment]
Reverted my changes from the links and moved them to a new branch to keep pull separate.

**Pull request merged**